src/components/wizard/WizardStep.jsx

```diff
@@ -64,73 +44,33 @@ export default class WizardStep extends React.Component {
   };
 
   onChange = (index, change) => {
-    this.context.updateFormQuestionsData(this.props.stepIndex || index, {
-      ...this.props.step,
+    this.context.updateFormQuestionsData(this.props.index || index, {
+      ...this.props.question,
       ...change,
     });
   };
 
-  _onMouseEnterHandler = () => {
-    this.setState({ showIcon: true });
-  };
-
-  _onMouseLeaveHandler = () => {
-    this.setState({ showIcon: false });
-  };
-
   render() {
-    const categoryClass = Question._getQuestionCategoryClass(this.props.step);
-    const question = this.context.getFormQuestionsData([this.props.stepIndex]);
-    const options = this.props.options;
-
-    let questionComponent = this.props.mapComponent(this.props.step, Question);
-    let questionElement = React.createElement(questionComponent, {
-      question: this.props.step,
-      onChange: this.onChange,
-      withoutCard: !FormUtils.isAnswerable(question),
-      index: this.props.stepIndex,
-    });
+    const question = this.context.getFormQuestionsData([this.props.index]);
 
     return (
-      <div className="wizard-step">
-        <Card className="wizard-step-content">
-          <Card.Header
-            className="bg-primary text-white question-header"
-            as="h6"
-            id={this.props.step["@id"]}
-            onMouseEnter={this._onMouseEnterHandler}
-            onMouseLeave={this._onMouseLeaveHandler}
-          >
-            {JsonLdUtils.getLocalized(
-              this.props.step[JsonLdUtils.RDFS_LABEL],
-              this.props.options.intl
-            )}
-            {QuestionStatic.renderIcons(
-              question,
-              options,
-              this.onCommentChange,
-              this.state.showIcon
-            )}
-          </Card.Header>
-          <Card.Body className={categoryClass}>{questionElement}</Card.Body>
-        </Card>
-
+      <React.Fragment>
+        <Question question={question} onChange={this.onChange} />
         {this.props.options.wizardStepButtons &&
           this._renderWizardStepButtons()}
-      </div>
+      </React.Fragment>
     );
   }
 }
 
-WizardStep.contextType = FormQuestionsContext;
-
 WizardStep.propTypes = {
-  options: PropTypes.object.isRequired,
-  step: PropTypes.object.isRequired,
```

Comment:
@LaChope why is `step` and `options` not required anymore?

Reply:
I completely forgot to put it back... Done now :)